command/CommandController.js

```diff
@@ -140,7 +140,7 @@ router.post('/article', VerifyToken, async function(req, res) {
       req,
```

Comment:
Let's add here in the code what the required and optional params are in the body for the /article API.  Even though we have this in the postman, it makes the code more readable to have it here.